data/campaigns/Eastern_Invasion/scenarios/06b_Soradoc.cfg

```diff
@@ -0,0 +1,1039 @@
+        {RAVANAL_ATTACK} {GENERIC_UNIT 4 (Wraith)  $possessee.x $possessee.y} {ANIMATE}
+
+        [delay]
+            time=250
+        [/delay]
+
+        {RAVANAL_DEPOSSESS 30 34}
+
+        #         [micro_ai]
+        #             side=4
+        #             ai_type=assassin
+        #             action=add
+        #             [filter]
+        #                 type=Wraith,Spectre
+        #             [/filter]
+        #             [filter_second]
+        #                 id=Dacyn
+        #             [/filter_second]
+        #         [/micro_ai]
+    [/event]
+
+    #
+    # Owaec meets up with Yannic
+    #
+    [event]
+        name=moveto
+        [filter]
+            id=Owaec
+        [/filter]
+        [filter_condition] # if Owaec moves adjacent to Yannic
+            [have_unit]
+                id=Yannic
+                [filter_adjacent]
+                    id=Owaec
+                [/filter_adjacent]
+            [/have_unit]
+        [/filter_condition]
+        [fire_event]
+            name=meet_yannic
+        [/fire_event]
+    [/event]
+
+    # if 3/4 enemy leaders die, that counds as having saved yannic too
+    [event]
+        name=die
+        [filter]
+            side=3,4,5,6 # for some reason 3-6 doesn't work
+            canrecruit=yes
+        [/filter]
+        [filter_condition]
+            [have_unit]
+                side=3,4,5,6 # for some reason 3-6 doesn't work
+                canrecruit=yes
+                count=1
+            [/have_unit]
+        [/filter_condition]
+
+        [message]
+            speaker=Gweddry
+            message= _ "That’s almost all the undead leaders defeated!"
+        [/message]
+        [fire_event]
+            name=meet_yannic
+        [/fire_event]
+    [/event]
+
+    [event]
+        name=meet_yannic
+        [filter_condition]
+            [have_unit]
+                id=Yannic
+            [/have_unit]
+        [/filter_condition]
+        [message]
+            speaker=Owaec
+            message= _ "Who commands this garrison? Time is short. We must evacuate what’s left of your men before—"
+        [/message]
+        [message]
+            speaker=Yannic
+            message= _ "Owaec? Is that you?"
+        [/message]
+        [message]
+            speaker=Owaec
+            message= _ "Yannic?! Well met friend, I haven’t seen you in years!! HA, you never could stay out of trouble! How’s your horse, clansman?"
+        [/message]
+        [message]
+            speaker=Yannic
+            message= _ "Injured, along with those few of my men lucky enough to yet live. I am more than grateful for your timely arrival and help."
+        [/message]
+        [message]
+            speaker=Dacyn
+            message= _ "Speaking of help, we require yours. I, the king’s High Advisor, seek a powerful magic east of the Listra to destroy these undead. We will require your men and provisions for the journey."
```

Comment:
Capitalise 'king'?

Reply:
Deliberate actually!  I'm not sure how consistent this is, but I didn't have Dacyn capitalize King to emphasize that he's not particularly respectful or reverent.  Gweddry and especially Owaec should always capitalize King.  Not a big deal either way though, especially if it's not consistent or if that point isn't getting across.

Reply:
It wasn't clear to me. It was clear that Dacyn doesn't respect Owaec nor Gweddry, but I missed the lack of respect for the Konrad. And even if this was the case, I don't think it comes through with the lack of capitalisation. I leave it to you and nemaara to decide what to do here, though.

Reply:
Just FYI according to most conventions that I know of, capitalization of these titles is before names or to replace usage of a proper noun, like a name. So if you were to say "I will do as you ask, Konrad!" it would become "I will do as you ask, my King!". But in this case, it's just used as a common noun so it doesn't get capitalized. I do need to go through and make sure it's correct elsewhere though.

Reply:
Yes, which is why I suggested the capitalisation, since Dacyn would otherwise say 'I, Konrad's High Adviser...'

Reply:
Why would capitalizing "king" give an implication of respect? Respect is shown by addressing the king as "Your Majesty" or "Sire", and referring to him in the third person as "His Majesty". I agree with nemaara that capitalizing "king" here would be wrong, and I do think referring to him as "the king" shows a lack of respect, so that's fine… but capitalizing "king" doesn't make sense to show respect. I think the only time you should capitalize "king" is when it's followed by the king's name.

> So if you were to say "I will do as you ask, Konrad!" it would become "I will do as you ask, my King!".

This should rather become "I will do as you ask, Sire!" or possibly "I will do as you ask, my liege!". I think the second one has slightly different implications though, so I'd probably recommend Sire most of the time. In particular, "my liege" only makes sense from someone who is the king's subject, whereas I believe "Sire" works no matter the speaker. (Also, "Your Majesty" could replace Sire, especially if it's the first time this person has addressed the king.)